python/paddle/trainer_config_helpers/data_sources.py

```diff
@@ -84,6 +84,7 @@ def py_data2(files, load_data_module, load_data_object, load_data_args,
             data.load_data_module = load_data_module
             data.load_data_object = load_data_object
             data.load_data_args = load_data_args
+            data.async_load_data = True
```

Comment:
always enable double buffer.

The `async load` is very confusing. It just enable double buffer in DataProvider.

Maybe we should change it later.